src/i3ipc.mli

```diff
@@ -133,6 +147,16 @@ module Reply : sig
     loaded_config_file_name: string;
   }
 
+  type binding_modes = string list
+
+  type config = {
+    config : string
+  }
+
+  type tick = {
+    tick_success : bool
+  }
```

Comment:
Do not expose this type as it is "internal" (the implementation directly extract the boolean).